kubeflow/registry.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: '0.1'
 kind: ksonnet.io/registry
 libraries:
   core:
-    version: master
+    version: issue_239_ambassador_as_proxy_for_jupyterhub
```

Comment:
FWIW this is what I do for debugging
https://github.com/kubeflow/kubeflow/blob/master/developer_guide.md